src/main/scala/org/broadinstitute/dsde/firecloud/service/UserService.scala

```diff
@@ -117,35 +117,25 @@ trait UserService extends HttpService with PerRequestCreator with FireCloudReque
         }
       }
     } ~
-    requireUserInfo() { userInfo =>
     pathPrefix("api") {
       path("profile" / "billing") {
         passthrough(UserService.billingUrl, HttpMethods.GET)
       } ~
       path("profile" / "billingAccounts") {
-        oauthParams { (state, _) => requestContext =>
+        get { requestContext =>
           val pipeline = authHeaders(requestContext) ~> sendReceive
-          val extReq = Get(UserService.billingAccountsUrl)
-          pipeline(extReq) onComplete {
-            case Success(response) if response.status == Forbidden =>
-              val tryParseScopes = Try {
-                response.entity.asString.parseJson.convertTo[ErrorReport].message.parseJson.convertTo[BillingAccountScopes]
-              }
-              tryParseScopes match {
-                case Success(scopes) =>
-                  // user does not have appropriate scopes.  Ask user to enable them via OAuth
-                  val redirectURI = HttpGoogleServicesDAO.getGoogleRedirectURI(state, "auto", Option(scopes.requiredScopes ++ HttpGoogleServicesDAO.userLoginScopes))
-                  requestContext.complete(Forbidden, ErrorReport(Forbidden, BillingAccountRedirect(redirectURI).toJson.toString))
-                case _ =>
-                  requestContext.complete(Forbidden, response.entity)
-              }
-            case Success(response) => requestContext.complete(response.status, response.entity)
-            case Failure(regrets) => respondWithErrorReport(regrets, requestContext)
+          pipeline(Get(UserService.billingAccountsUrl)) map { response =>
+            response.status match {
+              // The proxy would prevent a user from getting here if their account wasn't enabled,
+              // so a 403 from Rawls is effectively guaranteed to mean that the user hasn't enabled
+              // the billing accounts scope.
+              case StatusCodes.Forbidden =>
+                requestContext.complete(
+                  StatusCodes.Forbidden, Map("billingPermissionsRequired" -> true))
```

Comment:
I thought we discussed (via Slack) sending the error message from rawls - which contains the name of the missing scope - back to the user? Without that, API users have no way of knowing which scope is missing ... and if the back-end logic changes to require new/different scopes, we have no way of communicating that.

This PR has gone on for a long time and I guess I won't hold it up any longer on this point, but I'd prefer to pass the scope details to the user, instead of obfuscating them.